src/Components/design/Button/Button.stories.js

```diff
@@ -56,3 +56,9 @@ export const ButtonWithIcon = (props) => (
     </Button>
   </Container>
 );
+
+export const ButtonUserIcon = (props) => (
+  <Button {...props} variant={"usericon"} color={"colorwheel"} text={"JW"}>
```

Comment:
since these aren't dynamic we can omit the curlies
```suggestion
  <Button {...props} variant="usericon" color="colorwheel" text="JW">
```